### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by LokurasRlz**
# In this branch, I have worked on the following things: 

- Added Styles. ✔️ 
- Followed Zeplin's Design. ✔️
- Followed Linters. ✔️
- Followed GitFlow. ✔️
- Used the ducks pattern . ✔️

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @LokurasRlz , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors. Kindly fix them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+            <Link to="/">BOOKS</Link>
+          </li>
+          <li className="navCat">
+            <Link to="categories">CATEGORIES</Link>
+          </li>
+        </ul>
+      </div>
+      <div className="user">
+        <FaUser />
+      </div>
```
Comment (AmaduKamara) [inline src/components/NavBar.js:20]: - [ ] Kindly re-work the Navbar so the logo(Bookstore CMS) displays nicely and is not wrapped and also the icon spans to the right. See the screenshots below.

**The original design**
image

**Your Design**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Requires Changes ♻️♻️♻️ 

Hi @LokurasRlz,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅No linter error
✅ Good descriptive PR


## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly make the contents stay in the center of the page. Currently, the whole structure is left aligned. You can use a container style and center things up.

**The original app**
image

**Your design**
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by omarsalem7 — APPROVED** [review]
Hi @LokurasRlz 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**